.github/workflows/sf-security-check.yml

```diff
@@ -0,0 +1,14 @@
+name: PHP Symfony Security Check
+on: [ push, pull_request ]
```

Comment:
This should be done nightly, not on push/PR

Reply:
Thanks, I didn't agree at first but @Progi1984 convinced me by saying this was your request!! :trollface: